src/dialogs/list-diagrams/list-diagrams.tsx

```diff
@@ -42,7 +49,15 @@ export const ListDiagrams: React.FC<Props> = ({
   function getSubtitle(diagram: IDiagram) {
     const createdAt = getShortFormatByDate(diagram.createdAt);
     const updatedAt = getShortFormatByDate(diagram.updatedAt);
-    return `criado em: ${createdAt} - atualizado em: ${updatedAt}`;
+    return `Criado em: ${createdAt} - Última atualização: ${updatedAt}`;
+  }
+
+  function getType(diagram: IDiagram) {
+    return `Tipo de diagrama: ${diagram.type}`;
+  }
+
+  function checkIsDefault(diagram: IDiagram) {
+    return `${diagram.name} ${diagram.isDefault ? `- Padrão` : new String()}`;

```

Comment:
No lugar de `new String()`, colocar uma string vazia (ou apenas `String()`)

```suggestion
    return `${diagram.name} ${diagram.isDefault ? `- Padrão` : ''}`;

```